### ISSUE TITLE
Feature Request : Categorize application experiences based on tech stack.

### ISSUE THREAD

**Issue by s-ras** [mo1ein/jobname#8]
اکثر تجارب مطرح شده برای اپلای در بلاگ مربوط به بک‌اند و به خصوص golang هستند. با رشد پروژه و جذب تجارب افراد دیگه نیاز به دسته‌بندی تجربه‌ها بر اساس استک، زبان و موقعیت شغلی ایجاد می‌شه.

**Comment by mo1ein** [comment]
> اکثر تجارب مطرح شده برای اپلای در بلاگ مربوط به بک‌اند و به خصوص golang هستند. با رشد پروژه و جذب تجارب افراد دیگه نیاز به دسته‌بندی تجربه‌ها بر اساس استک، زبان و موقعیت شغلی ایجاد می‌شه.

Yes, I agree with you. If people contribute in other fields, this feature is useful. So far, no one has added experiences in other fields.
